package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@mux/elements",
   "engines": {
-    "node": "^14.17.0",
+    "node": "^14.17.0 || ^16.13.0",
     "npm": "^7.17.0"
```

Comment:
non-blocking: we should also update this npm version to be 8.x. It's what's ships with 16.x.